### ISSUE TITLE
_interopRequireDefault is not a function

### ISSUE THREAD

**Issue by imraazrally** [jdalrymple/node-gitlab-api#55]
I'm using node 6.11.0 locally. 
I'm requiring the module `require(node-gitlab-api/dist/es5)`

I'm getting the following error when running the app. This only occurs for **versions 2.2.5 and above**

Any help would be appreciated. 

```
var _classCallCheck2 = _interopRequireDefault(require("babel-runtime/helpers/classCallCheck"));
                       ^
TypeError: _interopRequireDefault is not a function
```

**Comment by jdalrymple** [comment]
hmm weird. ill look into it!! Sorry for the inconvience :(

**Comment by imraazrally** [comment]
Thanks!

**Comment by jdalrymple** [comment]
Think ive narrowed it down the node url library, since that was the only thing changed in that release! will fix today

**Comment by jdalrymple** [comment]
Think it might be fixed now. Give the latest (2.2.7) a run to check! (I was missing babel-runtime as a dependency)

**Comment by jdalrymple** [comment]
Did this change solve your problem? :)

**Comment by imraazrally** [comment]
Hey, sorry for the late response. I will test this tonight and confirm!  hope that’s fine. :)

**Comment by imraazrally** [comment]
I just tested this. I'm still running into the same issue. I did a console.log on the `_interopRequireDefault` method at **dist/es5/index.js**  and it looks like it's an object and not a function. however, it's being used as a function in the following lines. That doesn't was odd.

`
var _interopRequireDefault = require("babel-runtime/helpers/interopRequireDefault");
console.log(_interopRequireDefault)
var _API = _interopRequireDefault(require("./API"));
`

**Comment by jdalrymple** [comment]
hmm, that is weird. ill keep looking! Sorry for the false hope :(

**Comment by jdalrymple** [comment]
Alright, i think i got it to work this time. Tested locally and it seems to work, published in 2.2.8. Check it out once you get the chance!

**Comment by imraazrally** [comment]
Awesome. That fixed it!
